### ISSUE TITLE
Feature Request: marker for split_diff

### ISSUE THREAD

**Issue by akurz42** [surdu/scroll-marker#16]
Since minimap-split-diff makes my Atom painfully slow, I'd really appreciate some "lighter" diff highlighting. scroll-marker would be the ideal place for that!

Regards,
Alex

**Comment by surdu** [comment]
Yes, this could be implemented with scroll-marker, but it requires another feature first: the possibility to add a marker that spans more than one line.

Otherwise, could be implemented with today's features, but I don't believe it will look good.

I've added the new feature as an issue not to forget about it.

**Comment by akurz42** [comment]
Thanks for considering! From my perspective even a one-line indicator for a whole block would still be tremendously helpful as most of the times indicators like this are used to quickly locate differences and to check if anything was missed when scrolling manually.

Alex
